### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by tilgovi**
Automatically lint on commit using `prettier` integrated with `eslint`. Also, clean up our `eslint` config a bit to make it more strict and more accurate to our environment.

### CODE REVIEW

File: scripts/.eslintrc.json [hunk 1]
```diff
@@ -1,7 +1,13 @@
 {
+  "env": {
+    "node": true
```
Comment (BigBlueHat) [inline scripts/.eslintrc.json]: We should probably use `shared-node-browser` for `env`:
http://eslint.org/docs/user-guide/configuring#specifying-environments

We can discuss the audience of each package on the mailing list.

File: scripts/.eslintrc.json [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+  },
   "globals": {
     "__dirname": false,
     "__filname": false,
     "__moduleName": true
+  },
+  "rules": {
+    "no-console": 0
```
Comment (BigBlueHat) [inline scripts/.eslintrc.json]: Could we use the eslint "magic string" value of `"off"` here? It's clearer once we start mixing in `"warn"` and `"error"`: http://eslint.org/docs/user-guide/configuring#configuring-rules

File: scripts/serve.js [hunk 3]
```diff
@@ -39,19 +39,16 @@ const packages = fs.readdirSync(packagesPath);
 
 const app = express();
 
-packages.forEach((name) => {
+packages.forEach(name => {
   const root = path.join(packagesPath, name);
   const entry = path.join(root, 'index.js');
   const dest = path.join(root, `${name}.bundle.js`);
-  const external = (id) => /^@(annotator|hot)/.test(id);
+  const external = id => /^@(annotator|hot)/.test(id);
```
Comment (BigBlueHat) [inline scripts/serve.js]: Is the removal of parens from them `google` presets? I actually rather like having them around as I find hunting for the extra `>` in a line tedious.

But again...mailing list? :wink:

Comment (robertknight) [inline scripts/serve.js]: This is how prettier formats arrow functions with a single argument. There is [no option](https://github.com/prettier/prettier#options) to change this.

By design, prettier provides only a handful of options to choose major style options such as single vs. double quotes.

File: scripts/.eslintrc.json (hunk 1 above)
Comment (tilgovi) [inline scripts/.eslintrc.json]: This is the local .eslintrc.json for the scripts folder, container build scripts that run only in node.

File: scripts/serve.js (hunk 3 above)
Comment (BigBlueHat) [inline scripts/serve.js]: Works for me. Thanks for the explanation @robertknight

**Comment by tilgovi** [comment]
Merged with the change to use "off" instead of "0".

**Pull request closed without merging**